app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsActivity.java

```diff
@@ -50,20 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_event_details);
 
-        ButterKnife.bind(this);
+        Event initialEvent = getIntent().getParcelableExtra(EVENT_KEY);
+
+        if(initialEvent == null) {
```

Comment:
>If there comes a point where someone forgets to pass a parameter to event detail activity, would you want the app to crash?

yes, so that it can be fixed. Not that important, I just thought adjusting that case will make hard for developer to notice that bug.